app/components/atoms/paragraph/paragraph.component.tsx

```diff
@@ -9,8 +9,9 @@ interface Props {
 export const Paragraph = styled.p<Props>`
   color: ${({theme}) => theme.colors.black};
   font-family: ${({theme}) => theme.fonts.body};
-
   ${({bold}) => bold && 'font-weight: bold'};
   ${({italic}) => italic && 'font-style: italic'};
   ${({lineHeight}) => lineHeight && `line-height: ${lineHeight}`};
+  margin-block-start: 0;
```

Comment:
why is this needed?

Reply:
this would affect all `<Paragraph>` components FYI.

Reply:
Yes I know, I told Saúl and he tells me that he hasn't used the Paragraph yet. So it will not affect the created components